css/properties/-moz-user-modify.json

```diff
@@ -0,0 +1,72 @@
+{
+  "css": {
+    "properties": {
+      "-moz-user-modify": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/CSS/-moz-user-modify",
```

Comment:
This link 404s, I think you meant https://developer.mozilla.org/docs/Web/CSS/-moz-user-modify?